pkg/telemetryquerylanguage/tql/condition.go

```diff
@@ -18,13 +18,37 @@ import (
 	"fmt"
 )
 
-type CondFunc = func(ctx TransformContext) bool
+type condFunc = func(ctx TransformContext) bool
```

Comment:
This should still be exposed

Reply:
Also, should probably be renamed as it now conflicts with the "Condition" concept.